merlin/models/tf/blocks/dlrm.py

```diff
@@ -100,7 +101,23 @@ def DLRMBlock(
             "last layer of bottom MLP ({bottom_block.layers[-1].units}) "
         )
 
-    embeddings = EmbeddingFeatures.from_schema(cat_schema, embedding_options=embedding_options)
+    embeddings_kwargs = dict(
+        sequence_combiner=embedding_options.combiner,
```

Comment:
assuming `sequence_combiner` is the same thing as `combiner` on the EmbeddingOptions.